verifiable/credential.go

```diff
@@ -28,6 +36,258 @@ type W3CCredential struct {
 	RefreshService    *RefreshService        `json:"refreshService,omitempty"`
 }
 
+// VerifyProof verify credential proof
+func (vc *W3CCredential) VerifyProof(ctx context.Context, proofType ProofType, resolverURL string, opts ...StatusOpt) (bool, error) {
```

Comment:
I suspect that StatusOpt will not be only options in the future, it's better to rename to W3CProofVerifcationOpt

Reply:
renamed